docs/constraints/CSharp.md

```diff
@@ -34,7 +34,7 @@ These are the constraints that is applied to enum keys. The `Rule key` is what y
 |NO_NUMBER_START_CHAR|No numbers as starting characters|Default behavior is pre pending `number_` in front of the first character|
 |NO_EMPTY_VALUE|No empty values|Default behavior is to use `empty` as name. |
 |NO_RESERVED_KEYWORDS|No reserved keywords|C# has a list of reserved keywords ([see the full list here](../../src/generators/csharp/Constants.ts))|
-|NAMING_FORMATTER|Must be formatted equally|Model name is formatted using pascal case|
+|NAMING_FORMATTER|Must be formatted equally|Property name is formatted using pascal case|
```

Comment:
This one here is for the enum keys and how they are formatted, not the property name 🙂 Same applies for all the constraints.